src/middlewares/auth/authCustom.js

```diff
@@ -0,0 +1,63 @@
+const { User, UserStatuses, UserRoles } = require('../../models');
+const bcrypt = require('bcryptjs');
+const passport = require('passport');
+const CustomStrategy = require('passport-custom').Strategy;
+const GoogleStrategy = require('passport-google-oauth2').Strategy;
+require('dotenv').config();
+
+const verify = async (req, done) => {
+  const checkUser = await User.findOne({ email: req.body.email }).exec();
+
+  if (!checkUser) {
```

Comment:
Instead of 2 ifs you can write one: if(!user || user.status.toString() === status._id.toString()){...}

Reply:
Fixed.